src/main/resources/static/js/lastQuestionSlider.js

```diff
@@ -0,0 +1,22 @@
+function adapt(){
```

Comment:
good morning! that's what I was thinking yesterday and I've already thought of a solution that I'll push up to you here soon!
 

    On Thursday, July 7, 2016 9:27 AM, Dan Funk <notifications@github.com> wrote:
 

 In src/main/resources/static/js/lastQuestionSlider.js:> @@ -0,0 +1,22 @@
> +function adapt(){
Never repeat code. This function is identical to function in firstQuestionSlider. I would combine this code with the other slider.js file, rather than getting different behavior based on which of these files you include in your HTML. That can be confusing and hard to debug for people that come after you. It is great that you identified and are addressing the issue with BBSIQ. This is good first try.Rather than use an ID to find the last element, you can add a second class to the element. Just separate classes with a space. See http://www.avajava.com/tutorials/lessons/how-do-i-assign-multiple-classes-to-an-element.html
As I mention again later, an id should be unique across the whole document, so this won't work consistently, and shouldn't really work at all.—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.